charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -20,8 +20,8 @@ spec:
       annotations:
         kompose.cmd: kompose convert
         kompose.version: 1.21.0 (992df58d8)
-{{ if .Values.globals.appServiceAnnotations }}
-{{ toYaml .Values.globals.appServiceAnnotations | indent 4 }}
+{{ if .Values.services.apps.appServiceAnnotations }}
```

Comment:
Given the namespacing we not have under `services.app` we could shorten the variable name to just `annotations`  i.e. 

```
Values.services.apps.annotations
Values.services.proxy.annotations
Values.services.worker.annotations
```

This is a little easier to read and more consistent